packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/intel/ITIPIntelTest.java

```diff
@@ -121,6 +122,102 @@ public void testIpLookupMalicious_8() throws PangeaException, PangeaAPIException
         assertNotNull(response.getResult().getRawData());
     }
 
+    @Test
+    public void testIpGeolocate_1() throws PangeaException, PangeaAPIException {
+        // Default provider, not verbose by default, not raw by default;
+        IpGeolocateResponse response = client.geolocate("93.231.182.110");
```

Comment:
do we want to make `93.231.182.110` a class level variable so you don't have to change 12 locations if we ever have to change the IP?

Reply:
No sure about your question. In which class would so save the IP? in IpGeolocateResponse? I don't get the purpose of that